CMakeLists.txt

```diff
@@ -2626,9 +2622,9 @@ endif()
 
 # WavPack audio file support
 find_package(wavpack)
-option(WAVPACK "WavPack audio file support" OFF)
+cmake_dependent_option(BULK "WavPack audio file support" ON "wavpack_FOUND" OFF)
```

Comment:
Oops, I missed this error. I'll replace BULK with WAVPACK in another PR.

Reply:
Oh, copy & paste error :see_no_evil: